nexus/tests/integration_tests/vpc_firewall.rs

```diff
@@ -162,20 +170,6 @@ async fn test_vpc_firewall(cptestctx: &ControlPlaneTestContext) {
     .unwrap();
 }
 
-async fn get_rules(
-    client: &dropshot::test_util::ClientTestContext,
-    url: &str,
-) -> Vec<VpcFirewallRule> {
-    NexusRequest::object_get(client, url)
-        .authn_as(AuthnMode::PrivilegedUser)
-        .execute()
-        .await
-        .unwrap()
-        .parsed_body::<VpcFirewallRules>()
-        .unwrap()
-        .rules
-}
-
```

Comment:
The changes from here up in this file are noise — I got rid of `get_rules` because it was equivalent to an `object_get` one-liner.